### PR TITLE
fw driver fix for terraform issue 463

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[fw driver fix for terraform issue 463s]

### Issues Resolved
[List any issues this PR will resolve. e.g., Fixes #01]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: ov/firmware_drivers.go [hunk 1]
```diff
@@ -149,12 +149,15 @@ func (c *OVClient) GetFirmwareBaselineById(id string) (FirmwareDrivers, error) {
 
 func (c *OVClient) GetFirmwareBaselineByNameandVersion(name string) (FirmwareDrivers, error) {
 	var fwname, version string
-	fwNameVersion := strings.SplitAfter(name, ",")
-	if len(fwNameVersion) < 1 {
+	fwNameVersion := strings.Split(name, ",")
+	if len(fwNameVersion) == 0 {
```
Comment (AvijitOS) [inline ov/firmware_drivers.go:153]: do we have unit tests written considering different scenario where user has send empty, only name/only version or both.
If not can we test those scenarios manually now

Comment (AvijitOS) [inline ov/firmware_drivers.go:153]: I think this condition appear when we have empty string...as per doc "If s does not contain sep and sep is not empty, Split returns a slice of length 1 whose only element is s", have we taken care this scenario

Comment (nabhajit-ray) [reply ov/firmware_drivers.go:153]: Without separator the Split function will return only the string

Comment (nabhajit-ray) [reply ov/firmware_drivers.go:153]: Yes, it is tested.

If no string is provided(len==0) we are returning an error.
Also, if only version is provided( fwname=="") we are returning an error.

**Review by AvijitOS — APPROVED** [review]

**Pull request merged**